### ISSUE TITLE
Index issues

### ISSUE THREAD

**Issue by mzed** [immersive-audio-live/ADM-OSC#21]
How to deal with source/object numbers with multichannels sources.  

ADM compatibility

**Comment by mzed** [comment]
Names?

**Comment by madees** [comment]
Can you please describe a little bit more this issue ?

**Comment by mzed** [comment]
Yes.  A basic source/object might be one audio channel.  That's how L-ISA works, each source is mapped to one, mono audio input. However, there is the possibility to link multiple objects into a stereo object. Other environments might treat a whole bed as a single set.  This topic came up when discussing how ADM-OSC might make it easier to control stereo or multichannel "objects".  Rather than controlling sources /1 and /2, it might be easier to call them something like /stereogroup/1 or... ?  

Some PlugFest participants suggested this is a UI issue that should be handled there.  A particular control UI could present groups in the best way for the intended user, and generate the appropriate set of mono ADM-OSC messages based on some higher-level UI interaction.
